packages/mobile/.env.alfajores

```diff
@@ -1,7 +1,8 @@
 ENVIRONMENT=alfajores
 DEFAULT_TESTNET=alfajores
-# -1 == ZeroSync, 5 == Ultralight, see src/geth/consts.ts for more info
+# Sync mode when geth enabled, see src/geth/consts.ts for more info
 DEFAULT_SYNC_MODE=5
+ZERO_SYNC_INITIALLY=true
```

Comment:
Can you clarify what this config is for exactly? My first guess would be it's whether or not the app should use zero sync when it starts, but that would conflict with the DEFAULT_SYNC_MODE config we already have.

Reply:
DEFAULT_SYNC_MODE now specifies the sync mode for when geth is turned on
Because geth will turn on and off, DEFAULT_SYNC_MODE can't encode whether the app should use zeroSync initially. Added a comment in the .env file